doc/content2.tex

```diff
@@ -1,723 +0,0 @@
-
-        \begin{figure}[H]
-            \begin{center}
-                \scalebox{0.65}[0.5]{\includegraphics[angle=0]{img/src/results-overall-mse.pdf}}
-                \caption{Aggregated test results for all models. Average values of MSE for all tests samples.}
-                \label{img:results-overall-mse}
-            \end{center}
-        \end{figure}
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Conclusion} \label{chap:conclusion}
-The goal of the master's thesis is to develop independent module in Java language for providing alert prediction
-capabilities for open source project Hawkular.
-
-%The thesis first describes various approaches for time series forecasting, and evaluates which methods are suitable
-%for Hawkular requirements. Follows review of existing monitoring solutions and libraries for time series modelling.
-%It is shown that any of existing libraries are not suitable for Hawkular, therefore a decision was made to implement
-%selected models.
-
-Implementation part is split into multiple artifacts with dedicated functionality. The most important module
-focuses on time series modelling. It contains simple, double and triple exponential smoothing models which
-use non-linear optimization algorithm for finding the best parameters with goal of minimizing prediction error.
-The library also contains several time series utility classes and statistical tests for time series analysis.
-
-Data Mining prediction engine autonomously selects the best time series model for underlying time series. Most
-importantly, it is capable of dealing with concept drift problem. So in production there is no need of any
-analytical configuration.
-
-The last chapter benchmarks forecasting accuracy of the implemented models. It is shown that forecasts produced by
-implemented models are in some cases more accurate that forecast from package \texttt{forecast} from R language and
-in average Data Mining models can be compared to R models. In terms of execution time performance Data Mining
-models are much efficient.
-
-% TODO dobry cas? was?
-During the development of the module, Hawkular was still in development phase, therefore it was not
-possible to gather real data from production environment. This would help with tunning models for specific metrics
-for example Java heap usage.
-
-The module was successfully integrated into Hawkular and predictive charts of the metrics are located in Metric explorer
-tab.
-
-Future work on this module should analyze data from production servers and if necessary change the properties of
-optimizers in order to get more accurate predictions. Analysis could also contain evaluation if damped trend should be
-added to the implemented models.
-
-As it is open source project, it would be great to attract more people into development and continue with adding new
-models and features for time series analysis. Because in Java there is no widely used and maintained library.
```

Comment:
...analysis, because there is no widely used and maintained library in Java. /slovosled